Assets/Scripts/SettingsDataManager.cs

```diff
@@ -64,6 +76,9 @@ public SettingsData(SettingsDataStruct settingsData)
         SfxVolume = settingsData.SfxVolume;
         QualityPresetIndex = settingsData.QualityPresetIndex;
         DisplayModeIndex = settingsData.DisplayModeIndex;
+        // Settings added after this needs to be backwards compatible with players who saved old setttings
+        // This can be achieved by checking for larger than 0
+        CrosshairSize = settingsData.CrosshairSize > 0 ? settingsData.CrosshairSize : 1f;
```

Comment:
You um... forgot to use the default constant here :)))))

Reply:
Fixed 👍